src/topology/quasi_separated.lean

```diff
@@ -107,10 +117,10 @@ lemma quasi_separated_space.of_open_embedding (h : open_embedding f) [quasi_sepa
 (is_quasi_separated_univ α).mp
   (h.is_quasi_separated_iff.mpr $ is_quasi_separated.of_quasi_separated_space _)
 
-instance topological_space.compact_opens.has_inf_of_quasi_separated_space
-  [quasi_separated_space α] : has_inf (topological_space.compact_opens α) :=
+instance compact_opens.has_inf_of_quasi_separated_space
+  [quasi_separated_space α] : has_inf (compact_opens α) :=
```

Comment:
I think that's basically THE condition for `compact_opens` to have an inf, right?
```suggestion
instance [quasi_separated_space α] : has_inf (compact_opens α) :=
```

Reply:
The auto-generated name clashes with `instance [t2_space α] : has_inf (compact_opens α)` in `topology.sets.compacts` though. Should I move the new instance to that file and replace the t2 one instead?

Reply:
Does `t2_space` imply `quasi_separated_space`? Either way, this instance should be in `topology.sets.compacts`, yes.